JMdictFiles/JMdict_to_JSON.py

```diff
@@ -0,0 +1,51 @@
+import xml.etree.ElementTree as ET
```

Comment:
Please add module docstring in the first line of the file, explaining the need and usage of this script.